File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/DigitsOfANumber.java
import java.util.*;
public class DigitsOfANumber {
    public static void main(String[] args) {
        // write your code here  
         Scanner sc = new Scanner(System.in);
          int n= sc.nextInt();
          sc.close();
          int t,d,c=0;
          t=n;
          while(n>0)
          {
              c++;
              n=n/10;
          }
          while(c>=1||t>0)
          {
              d= t /(int) Math.pow(10,c-1);
              System.out.println(d);
              t= t % (int) Math.pow(10,c-1);
              c--;
            
          }
       }
}

File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/GettingStarted/digitsofanumber.cpp
#include <iostream>
#include<cmath>
using namespace std;

int main(){
    int n;
    cin >> n;
    int num=0;
    int p=n;
    while(p!=0){
        p=p/10;
        num=num+1;
        
    }
    int div = pow(10,num-1);
    while(div!=0){
        int q =n/div;
        int r=n%div;
        cout<<q<<endl;
        n=r;
        div=div/10;
    }

    //write your code here
    
}
